includes/template-library/sources/local.php

```diff
@@ -818,7 +864,8 @@ public function import_template( $name, $path ) {
 			for ( $i = 0; $i < $zip->numFiles; $i++ ) {
 				$zipped_file_name = $zip->getNameIndex( $i );
 				$zipped_extension = pathinfo( $zipped_file_name, PATHINFO_EXTENSION );
-				if ( 'json' === $zipped_extension ) {
+				// Template Kit zip files contain a `manifest.json` file, this is not a valid Elementor template so ensure we skip it.
+				if ( 'manifest.json' !== $zipped_file_name && 'json' === $zipped_extension ) {
```

Comment:
Flip the conditions, the `manifest.json` should be checked only for json files.

Reply:
Thanks @matipojo - feedback addressed :) https://github.com/elementor/elementor/pull/11316/commits/afefd3c557d0b6b0236d5906a5b6ac52d6e4012c